tests/scheduledUpdater.test.js

```diff
@@ -0,0 +1,23 @@
+import Scheduler from '../src/scheduledUpdater';
+
+jest.mock('request-promise', () => () => Promise.resolve(JSON.stringify({
+  access_token: 'token',
+  exp: 8400,
+})));
+
+describe('Scheduled updater', () => {
+  beforeAll(() => {
+    Scheduler.sendTokenNotificationEmail = jest.fn();
+    Scheduler.updateConfigOnHeroku = jest.fn();
+  });
+  it('should update Heroku with the new token', async () => {
```

Comment:
Let there consistently be linespaces, between test cases

Reply:
done